### PR TITLE
Initial add of Track2 ImageAnalysis SDK

### PR DESCRIPTION
**Pull request by rhurey**
Draft addition of ImageAnalysis SDK from https://github.com/Azure/azure-rest-api-specs/pull/26146

Marking as ready to get the pipeline run...

### CODE REVIEW

**Comment by rhurey** [comment]
@pallavit not specifically, we've been talking with Laurent about the process in general, but beyond arch board no one specifically has been looking at the changes.

What do we need to do to move this forward, now that the service has stabilized?

**Comment by pallavit** [comment]
> @pallavit not specifically, we've been talking with Laurent about the process in general, but beyond arch board no one specifically has been looking at the changes.
> 
> What do we need to do to move this forward, now that the service has stabilized?

@rhurey I believe Laurent is working with you on the next steps.  @christothes can help review the PR. Please note you will need to work with archboard on API shape approvals etc.

File: sdk/vision/Azure.AI.Vision.ImageAnalysis/README.md
Comment (christothes) [reply sdk/vision/Azure.AI.Vision.ImageAnalysis/README.md:104]: same in all the other snippets that include this comment

File: sdk/vision/Azure.AI.Vision.ImageAnalysis/README.md
Comment (christothes) [inline sdk/vision/Azure.AI.Vision.ImageAnalysis/README.md:156]: consistency with the above snippet (and more concise)
```suggestion
            Console.WriteLine($"     Word: '{word.Text}', Confidence {word.Confidence.Confidence:F4}, Bounding Polygon: [{string.Join(" ", word.BoundingPolygon)}]");
```

File: sdk/vision/Azure.AI.Vision.ImageAnalysis/README.md
Comment (christothes) [inline sdk/vision/Azure.AI.Vision.ImageAnalysis/README.md:163]: nit: I would re-word this to stand alone without reference to the previous snippet

File: sdk/vision/Azure.AI.Vision.ImageAnalysis/README.md
Comment (christothes) [inline sdk/vision/Azure.AI.Vision.ImageAnalysis/README.md:181]:
```suggestion
            Console.WriteLine($"     Word: '{word.Text}', Confidence {word.Confidence:F4}, Bounding Polygon: [{string.Join(" ", word.BoundingPolygon)}]");
```

File: sdk/vision/Azure.AI.Vision.ImageAnalysis/README.md
Comment (christothes) [inline sdk/vision/Azure.AI.Vision.ImageAnalysis/README.md:189]:
```suggestion
When you interact with Image Analysis using the .NET SDK, errors returned by the service correspond to the same HTTP status codes returned for REST API requests. For example, if you try to analyze an image that is not accessible due to a broken URL, a `400` status is returned, indicating a bad request.
```

File: sdk/vision/Azure.AI.Vision.ImageAnalysis/README.md
Comment (christothes) [inline sdk/vision/Azure.AI.Vision.ImageAnalysis/README.md:207]: would the `Message` property already contain an error similar to this? It seems odd to write your own 400 response error detail.

File: sdk/vision/Azure.AI.Vision.ImageAnalysis/README.md
Comment (christothes) [inline sdk/vision/Azure.AI.Vision.ImageAnalysis/README.md:219]: Is `Azure.AI.Vision.ImageAnalysis` the actual name of the service?

File: sdk/vision/Azure.AI.Vision.ImageAnalysis/README.md
Comment (christothes) [inline sdk/vision/Azure.AI.Vision.ImageAnalysis/README.md:235]: main branch

File: sdk/vision/Azure.AI.Vision.ImageAnalysis/samples/Sample01_HellowWorld.md
Comment (christothes) [inline sdk/vision/Azure.AI.Vision.ImageAnalysis/samples/Sample01_HellowWorld.md:1]: This looks like the readme format - usually this would just contain the specific sample snippet and associated descriptions.

Comment (christothes) [reply sdk/vision/Azure.AI.Vision.ImageAnalysis/samples/Sample01_HellowWorld.md:1]: same for the other sample md files

File: sdk/vision/Azure.AI.Vision.ImageAnalysis/src/Custom/ImageAnalysisOptions.cs
Comment (christothes) [inline sdk/vision/Azure.AI.Vision.ImageAnalysis/src/Custom/ImageAnalysisOptions.cs:37]: capitalize and convert to properties

File: sdk/vision/Azure.AI.Vision.ImageAnalysis/src/Custom/ImageBoundingBox.cs
Comment (christothes) [inline sdk/vision/Azure.AI.Vision.ImageAnalysis/src/Custom/ImageBoundingBox.cs:4]: delete?

File: sdk/vision/Azure.AI.Vision.ImageAnalysis/src/Custom/ImageBoundingBox.cs
Comment (christothes) [inline sdk/vision/Azure.AI.Vision.ImageAnalysis/src/Custom/ImageBoundingBox.cs:6]: remove?

File: sdk/vision/Azure.AI.Vision.ImageAnalysis/tests/Azure.AI.Vision.ImageAnalysis.Tests.csproj
Comment (christothes) [inline sdk/vision/Azure.AI.Vision.ImageAnalysis/tests/Azure.AI.Vision.ImageAnalysis.Tests.csproj:28]: we shouldn't need this

File: sdk/vision/Azure.AI.Vision.ImageAnalysis/tests/ImageAnalysisTestBase.cs
Comment (christothes) [inline sdk/vision/Azure.AI.Vision.ImageAnalysis/tests/ImageAnalysisTestBase.cs:62]: Still need to do this?

File: sdk/vision/Azure.AI.Vision.ImageAnalysis/tests/ImageAnalysisTestEnvironment.cs
Comment (christothes) [inline sdk/vision/Azure.AI.Vision.ImageAnalysis/tests/ImageAnalysisTestEnvironment.cs:25]: You may still want to have this come from the environment and default to `EastUs` if not specified

**Comment by pallavit** [comment]
@rhurey could you please also add an entry to the codeowners file - https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS to include the new SDK. We usually ask to include two people from the team so it would be great if you could add one other person (other than you) to that list.  This way you can get updates to the changes.

File: sdk/vision/Azure.AI.Vision.ImageAnalysis/CHANGELOG.md
Comment (christothes) [inline sdk/vision/Azure.AI.Vision.ImageAnalysis/CHANGELOG.md:10]: Should we add a link to the two rest api docs mentioned in these bullets?

File: sdk/vision/Azure.AI.Vision.ImageAnalysis/tests/Azure.AI.Vision.ImageAnalysis.Tests.csproj
Comment (christothes) [inline sdk/vision/Azure.AI.Vision.ImageAnalysis/tests/Azure.AI.Vision.ImageAnalysis.Tests.csproj:12]: I think VS adds this - but these directories shouldn't exist anymore. I think these can be manually removed.

File: sdk/vision/Azure.AI.Vision.ImageAnalysis/README.md
Comment (christothes) [inline sdk/vision/Azure.AI.Vision.ImageAnalysis/README.md:236]: remove all the duplicate stuff here that is already found in the main readme.md (same for other samples)

**Review by christothes — APPROVED** [review]

**Pull request merged**
